addons/website_mail_channel/static/src/js/website_mail_channel.editor.js

```diff
@@ -8,32 +8,68 @@ var wUtils = require('website.utils');
 var _t = core._t;
 
 options.registry.subscribe = options.Class.extend({
-    select_mailing_list: function (previewMode, value) {
+    /**
+     * @override
+     */
+    cleanForSave: function () {
+        this.$target.addClass('d-none');
+    },
```

Comment:
I know you just moved it but is this even needed? Is not there a related public widget whose destroy is doing this?

Reply:
Not right now, it is just the public widget start [removing the d-none](https://github.com/odoo/odoo/blame/193040ada2cd8e94a04383bbfea7f56a1a19340e/addons/website_mail_channel/static/src/js/website_mail_channel.snippet.js#L22)
I guess we should refactor it..